style.css

```diff
@@ -22,71 +21,508 @@ ul {
+  gap: 400px;
+}
+
+.line {
+  border: 1px solid #dfe1e6;
+  margin: 20px;
+}
+
+.framework {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  justify-content: space-between;
+  gap: 400px;
+}
+
+.other_skills {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  justify-content: space-between;
+  gap: 400px;
+}
+
+.skills ul {
+  display: flex;
+  flex-direction: row;
+}
+
+.skills li {
+  background-color: #f7f7f9;
+  margin: 15px;
+  padding: 30px;
+}
+
+.skills li p {
+  width: 98px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.contact_me {
+  width: 1440px;
+  height: 846px;
+  left: 0.001px;
+  top: 5125px;
+  background: #6070ff;
+  border-top-left-radius: 70px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  background-image: url("/images/contactform.png");
+}
+
+.contact_me_container {
+  font-style: normal;
+  color: #ffff;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.contact_title {
+  width: 801px;
+  height: 56px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #ffff;
+}
+
+.contact_des {
+  width: 569px;
+  height: 102px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact_form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  padding: 10px;
+}
+
+input {
+  height: 48px;
+  width: 447px;
+  padding: 10px;
+  border-radius: 2px;
+}
+
+textarea {
+  height: 48px;
+  width: 447px;
+  border-radius: 2px;
+}
+
+.btn_touch {
+  width: fit-content;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  border-radius: 8px;
   background-color: #ffff;
 }
 
-@media only screen and (max-width: 600px) {
-  * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+@media screen and (max-width: 480px) {
+  nav button {
+    display: inline-block;
+    background: transparent;
+    font-size: 1.8rem;
+    cursor: pointer;
+    color: #2230d2;
   }
 
-  .nav-links {
-    display: none;
+  header {
+    background-color: #ffff;
+    width: 375px;
   }
 
   .nav-bar {
-    display: grid;
-    grid-template-columns: 4fr 0.5fr;
+    gap: 200px;
     padding-top: 26px;
     padding-left: 24px;
-    font-family: "Poppins", sans-serif;
     font-style: normal;
     font-weight: 700;
     font-size: 18px;
     line-height: 20px;
   }
 
+  .nav-links {
+    display: none;
+  }
+
+  .intro {
+    width: 375px;
+    height: 718px;
+    left: 0.001px;
+    top: 0.001px;
+    border-bottom-left-radius: 70px;
+    padding-bottom: 60px;
+  }
```

Comment:
- [ ] As the previous reviewer mentioned, kindly please fix the width of the screen to match the Figma template.
- your design 
![image](https://user-images.githubusercontent.com/53402600/171746054-cd551f27-44a5-45eb-8943-b09e31e7d69f.png)

- [ ] [OPTIONAL] You are using fixed units for width `px`, I recommend you not give fixed units to not make extra spaces in your design like the above image. You can refer to this [guideline](https://web.dev/responsive-web-design-basics/) to make your design responsive and I recommend using flexbox and grid to lay out the elements on the page.